src/java/org/apache/cassandra/service/accord/AccordCommandStore.java

```diff
@@ -321,7 +315,7 @@ public long latestEpoch()
     }
 
     @Override
-    public Timestamp maxConflict(Keys keys)
+    public Timestamp preaccept(TxnId txnId, Keys keys)
```

Comment:
Why does `preaccept` not include the same logic as the implementation in `InMemoryCommandStore` around sometimes returning a new `Timestamp`?